packages/inferno-create-element/src/index.ts

```diff
@@ -16,9 +16,9 @@ const componentHooks = {
  * @param {string|Function|Component<any, any>} type Type of node
  * @param {object=} props Optional props for virtual node
  * @param {...{object}=} _children Optional children for virtual node
- * @returns {VNode} new virtual ndoe
+ * @returns {VNode} new virtual node
  */
-export function createElement<T>(type: string | Function | Component<any, any>, props?: T & Props<T> | null, _children?: any): VNode {
+export function createElement<T>(type: string | Function | Component<any, any>, props?: T & Props<T> | null, ..._children: any[]): VNode {
```

Comment:
Because current resulting d.ts only lets one pass zero items or a single one, errors on multiple. It'd become apparent sooner if more tests were type-checked.

![image](https://user-images.githubusercontent.com/20999066/47607635-fc3ae580-da2a-11e8-80b3-1994346f67b5.png)

I'll make a polymorphic definition instead, should keep current logic and result in working types.

Reply:
awesome :)